lib/modules/command.go

```diff
@@ -0,0 +1,30 @@
+package modules
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	i3barjson "github.com/davidscholberg/go-i3barjson"
+)
+
+type Command struct {
+	BlockConfigBase `yaml:",inline"`
+	Cmd             string `yaml:"command"`
+}
+
+func (c Command) UpdateBlock(b *i3barjson.Block) {
+	b.Color = c.Color
+	fullTextFmt := fmt.Sprintf("%s%%s", c.Label)
+
+	cmdParse := strings.Fields(c.Cmd)
+	cmd, args := cmdParse[0], cmdParse[1:]
+
+	if out, err := exec.Command(cmd, args...).Output(); err != nil {
+		b.Urgent = true
+		b.FullText = fmt.Sprintf(fullTextFmt, err.Error())
+		return
+	}
+
+	b.FullText = fmt.Sprintf(fullTextFmt, strings.Replace(string(out), "\n", "", -1))
```

Comment:
Also, I forgot to mention that this gives me a compile error:

```
# github.com/mroote/goblocks/lib/modules
lib/modules/command.go:29: undefined: out
```

This appears to be due to the `out` variable being declared in the if block, meaning that it's not available outside the block's scope. You can resolve this by just moving the `exec.Command` call before the if block, which will declare the `out` variable in the proper scope.